eventActions/accountabilityActions.js

```diff
@@ -1,5 +1,17 @@
 const config = require('../config.json');
 
+// Helper method to loop through pins. I was getting mad when I named it...
+function isMessagePinnedAtAll(messageToCheck, setOfPinnedMessages){
+	var fetchedMessagesIterator = setOfPinnedMessages.values();
+	var msgVal = fetchedMessagesIterator.next().value;
```

Comment:
Use const or let instead of var

Reply:
(in general, this should be fixed)

Reply:
the `msgVal` field changes, but iterator is now `const` 👌